pysoundcomparisons/__main__.py

```diff
@@ -491,17 +484,15 @@ def write_modified_soundfiles(args):
 
     api = _api(args)
 
-    catalog_items = {}
     return_data = {}
     return_new = set()
     return_modified = {}
     return_obsolete = {}
     return_check = {}
-    cdstar_object_metadata = {}
     valid_soundfilepaths = []
     server_md5_items = set()
 
-    valid_soundfilepaths_filepath = api.repos.joinpath(
+    valid_soundfilepaths_filepath = api.repos.joinpath('pysoundcomparisons',
```

Comment:
I would recommend not to write output to the package directory. At some point we may decide to move the package to its own repository, or to move it into a `src` directory, or the package may have been installed not in development mode.

I think any location in the repository other than `pysoundcomparisons` would be a better choice here. What about `soundfiles`?

Reply:
`soundfiles` sounds feasible